pi-hole/Dockerfile

```diff
@@ -93,6 +95,9 @@ RUN \
 # Copy root filesystem
 COPY rootfs /
 
+# Download the current root hints file (the list of primary root servers which are serving the domain "." - the root domain). 
+RUN wget -O /etc/unbound/root.hints https://www.internic.net/domain/named.root
```

Comment:
There should be a check once in a while if the root hints are changed.

Reply:
This is actually already done via crontab here:

https://github.com/hassio-addons/addon-pi-hole/pull/67/files#diff-ef028a23c37ee193f725283354e6653cR6

Reply:
Sorry I totally looked over the cron! 👎